tools-local/tasks/mobile.tasks/AndroidAppBuilder/ApkBuilder.cs

```diff
@@ -257,7 +257,7 @@ private static string GetLatestBuildTools(string androidSdkDir)
         string? buildTools = Directory.GetDirectories(Path.Combine(androidSdkDir, "build-tools"))
             .Select(Path.GetFileName)
             .Where(file => !file!.Contains("-"))
-            .Select(file => Version.TryParse(Path.GetFileName(file), out Version? version) ? version : default)
+            .Select(file => Version.TryParse(Path.GetFileName(file), out Version? version) ? (version ?? default) : default)
```

Comment:
The original code looks correct. Is this working around a Roslyn bug?

Reply:
it does look like a Roslyn issue yes since TryParse is annotated like this: `[NotNullWhen(true)] out Version? result`. Unless it's somehow building against a ref that doesn't that annotation?

Reply:
I have opened https://github.com/dotnet/roslyn/issues/46461 on this